### ISSUE TITLE
[Top Bar App]: [Objekt ID wird ab Element 9 wieder auf 0 gesetzt]

### ISSUE THREAD

**Issue by frank2604** [Scrounger/ioBroker.vis-materialdesign#212]
Hi,

ich wollte ein zehntes Navigationsleisten-Element hinzufügen, also [9].

Beim Umschalten auf diesen Menüpunkt wird die Objekt ID nur kurz auf 9 gesetzt, dann aber sofort auf 0 zurückgestellt.

**Versions:**  
 - Adapter version: 0.5.9
 - JS-Controller version: 14.18.1
 - VIS Adapter version: 1.4.15

**Comment by smarthomegirl** [comment]
Ich hatte das gleiche Problem.
Konnte es wie folgt lösen:

In dem "View in Widget8" dürfen keine Leerviews angegeben werden, so wie früher für ein Untermenü.

**Comment by dirni** [comment]
Konntest du das Problem lösen ich habe Aktuell das gleiche Problem

**Comment by frank2604** [comment]
Ja, ich habe mit 2 Datenpunkten und einem kleinen Skript einen Workaround gebaut.

Ich habe das gerade mal als Beitrag im Forum mit Screenshots beschrieben:

[https://forum.iobroker.net/topic/29660/material-design-widgets-top-app-bar-widget/402](url)

**Comment by dirni** [comment]
Ok ich habe es auch gelöst , ich habe den Datenpunkt neu angelegt und nun geht es
